apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -72,6 +72,9 @@ class OrderBlocksAnswerData(TypedDict):
 FIRST_WRONG_TYPES = frozenset(
     [FeedbackType.FIRST_WRONG, FeedbackType.FIRST_WRONG_VERBOSE]
 )
+RANKING_DAG_ORDERED_GRADING_TYPE = frozenset(
+    [GradingMethodType.RANKING, GradingMethodType.DAG, GradingMethodType.ORDERED]
+)
```

Comment:
I don't like the name of this array, can we use something more descriptive, like 'LCS_GRADABLE_TYPES' or something?

Reply:
fixed in [955ce68](https://github.com/PrairieLearn/PrairieLearn/pull/9292/commits/955ce68fe1b5f068d86c76cf1eae3e1101918dfd).